pkg/kubelet/server/server_test.go

```diff
@@ -204,7 +204,13 @@ type serverTestFramework struct {
 	criHandler      *utiltesting.FakeHandler
 }
 
-func newServerTest() *serverTestFramework {
+func newServerTest(disableDebug ...bool) *serverTestFramework {
```

Comment:
Why is `disableDebug` a slice?

Reply:
What I want to do is adding disableDebug with default value 'true' to the func newServerTest, then there would be no need to change other places. 
But golang do not support default value.
I don't want to change other places from newServerTest() to newServerTest(true), and the slice make the inputs "optional".

Reply:
Changeing newServerTest() to newServerTest(true) would do much harm, I think.

Reply:
There're 27 places, and possible most of these places do not actually care about the flag value.

Reply:
Haha, OK.
Let me ask another question:
Why disableDebug is a slice of bool? Why not a slice of struct{}

Reply:
because I still want it indicate a "flag". slice is just a way to make "optional".

To be fair, I do not really understand why golang do not support default value.

Reply:
The way this is typically done in go is to add a new method with the parameter, and delegate to it from the old, i.e.
```go
func newServerTestWithDebug(enableDebugging bool) *serverTestFramework {
...
}

func newServerTest() *serverTestFramework {
  return newServerTestWithDebug(true)
}
```

Reply:
o! Thanks a lot for this! Let me update the PR.